apps/ios/ios/FluentUIReactNativeTester_iOS-Bridging-Header.h

```diff
@@ -0,0 +1,3 @@
+//
+//  Use this file to import your target's public headers that you would like to expose to Swift.
```

Comment:
Do we actually need a bridging header?

Reply:
For an app target, bridging headers are safe, but agreed there's not a ton of value to an empty one unless we have immediate plans to put headers into it.